oak-core-spi/src/main/java/org/apache/jackrabbit/oak/spi/lock/LockConstants.java

```diff
@@ -16,14 +16,15 @@
  */
 package org.apache.jackrabbit.oak.spi.lock;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.Set;
 
-import com.google.common.collect.ImmutableSet;
 import org.apache.jackrabbit.JcrConstants;
 
 public interface LockConstants extends JcrConstants {
 
-    Set<String> LOCK_PROPERTY_NAMES = ImmutableSet.of(
-            JCR_LOCKISDEEP,
-            JCR_LOCKOWNER);
+    Set<String> LOCK_PROPERTY_NAMES = Collections
```

Comment:
I believe the goal is to upgrade `guava` not remove it. So we can keep using it, instead of moving this to Java, wdyt ?
cc @mreutegg

Reply:
I believe we should avoid use of Guava where we can; maybe at some later point of time we might decide to remove it.